src/transferlistwidget.cpp

```diff
@@ -741,6 +754,10 @@ void TransferListWidget::displayListMenu(const QPoint&)
     connect(&actionForce_recheck, SIGNAL(triggered()), this, SLOT(recheckSelectedTorrents()));
     QAction actionCopy_magnet_link(QIcon(":/Icons/magnet.png"), tr("Copy magnet link"), 0);
     connect(&actionCopy_magnet_link, SIGNAL(triggered()), this, SLOT(copySelectedMagnetURIs()));
+
+    QAction actionCopy_caption_list(tr("Copy name"), 0);
```

Comment:
Oh, the "wrongness" is just that the line endings are going to be in Unix format. This is actually OK in Windows since all text editors in Windows (except perhaps very old legacy software) are fully capable of dealing with Unix line endings.

I will leave this enhancement for some other day. It's trivial in my opinion, and doesn't cause any runtime errors in Windows.